tests/conftest.py

```
@@ -3,4 +3,4 @@
 
 @pytest.fixture(autouse=True)
 def settings_configuration(settings):
-    settings.INSTALLED_APPS += ["swo_playground.apps.ExtensionConfig"]
+    settings.INSTALLED_APPS = settings.INSTALLED_APPS + ["swo_playground.apps.ExtensionConfig"]  # noqa: PLR6104, RUF005
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `PLR6104`)

Remove unused `noqa` directive

(RUF100)